### ISSUE TITLE
Accessibility: exhibit creators need edit option for image titles in widgets

### ISSUE THREAD

**Issue by caaster** [projectblacklight/spotlight#3599]
### Issue Description - WCAG standard
I consulted with the Stanford Office of Digital Accessibility (SODA) today. I asked them if it was an accessibility violation for more than one image on the same exhibit page to have the same title, even if the alt text is different for each image. They said it was problematic and the titles should be different, but acknowledged that other accessibility issues would be higher priority to resolve.

### Method of discovery
Siteimprove, Wave, keyboard navigation, VoiceOver etc.
The examples I prepared for SODA consisted of 2 different exhibit feature pages (this is an unpublished Stanford exhibit):

- One feature page contained 3 different images with the same title ("Hopkins Marine Station"), all displayed using a single Item Slideshow widget. I created different, appropriate alt text for each image.
- One feature page contained 3 different images with the same title ("Hopkins Marine Station"), all displayed using a single Item Row widget. I created different, appropriate alt text for each image.

### Suggested solution
If a substitute is found for Sir Trevor, make sure that the image titles for widgets are editable. Here at Stanford (and maybe other institutions) the title is pulled from the metadata in the institutional repository and it is not editable. 

NOTE: With millions of items in most institutional repositories, it is not scalable to make sure each digital resource has a unique title.
